vuepress/docs/next/tutorials/samples/customer-portal.md

```diff
@@ -1,125 +1,166 @@
-* Role-Based Access Control (RBAC)  Entando Identity Management System
-* Integrated User, customer, project and subscription management 
+* Customizable service ticket and tracking system with Jira Service Management
+* Role-based access control (RBAC) with Entando Identity Management System
+* Integrated user, customer, project and subscription management 
 
-This tutorial covers 
+This tutorial covers: 
 1. [Installation](#installaton)
-2. [Configuration for Administrators](#configuration)
-3. [A brief User Reference](#user-refernce)
-
-# Prerequisites
-* A working instance of Entando running on Kubernetes. See [Getting Started](../../docs/getting-started/)for more information.
-* Use the Entando CLI to verify that all dependencies are installed:
-ent check-env develop
+2. [Configuration](#configuration)
+4. [Managing the Customer Portal](#manage-the-customer-portal)
+5. [Using the Customer Portal](#using-the-customer-portal)
+6. [Application Details](#application-details)
+7. [Resources](#resources)
+
+## Installation
+### Prerequisites
+* A working instance of Entando running on Kubernetes. See [Getting Started](../../docs/getting-started/) for more information or [install Entando on any Kubernetes provider](../tutorials#operations). 
+* Use the Entando CLI command `ent check-env develop` to verify all dependencies
 * Docker installed and running
 
-# Installation
-## Quick Install from Entando Hub
+### Automatic Install with App Builder
+Install the Customer Portal in a few simple steps from the Entando App Builder. 
+1. Go to the Repository from the left navigation bar, enter Entando Hub in the Registry field at the top of the page. You will now have access to the Hub.
 
-1. From the Entando Hub, click on the Customer Portal bundle from the Catalog
-2. Click Install to see a popup page with two commands. These need to be implemented in the order shown. Copy the first command and paste into your CLI. 
-3. Copy the second command and do the same. The content bundle will now be deployed.
-4. Go to the Entando App Builder and navigate to the Repository.  Here you will see the two bundles for the Portal. Install both bundles in the version you desire. You will be able to update the version at any time.
-5. To navigate to your Customer Portal: <br>
-from the left menu →  `Page` → `Mangement` <br>
-find `Customer Portal` under `Page Tree` list<br>
-from the `Actions` pull-down menu →  `View Published Page`
+2. `Install` the two bundles for the Customer Portal from the Catalog. Once completed, it is ready for use. Update the bundle version at any time. 
 
-## Manual Install 
-Top install the Entando Customer Portal manually to your specific environment, go to the [Entando Github Repository](https://github.com/entando-samples/customer-portal/tree/main/application) and refer to the README.MD there for more details.
+To go directly to your Portal, navigate to Pages Management, use the Actions for the Customer Portal and select View Published Page.
```

Comment:
after reading through this in its entirety i think we need to set some rules for how to treat terms, e.g. select View Published Page vs select "View Published Page" vs select 'View Published Page'